schemas.json

```diff
@@ -13,6 +13,8 @@
     },
     "Entity": { "type": "object" },
     "Fingerprint": { "type": "object" },
+    "QualifierHash": { "type": "object" },
```

Comment:
See comment above

Reply:
Yes, copy paste mistake

Reply:
but it's still an object, it should be a string, no?

Reply:
I thought they were all objects. Will change to string.